.github/workflows/psp-reusable.yml

```diff
@@ -13,7 +16,9 @@ on:
         required: true
 
 env:
-  artifact_name: build-${{ inputs.os }}-${{ inputs.build_script }}-${{ inputs.build_type }}
+  artifact_name: build-${{ inputs.os }}-${{ inputs.compiler }}-${{ inputs.build_script }}-${{ inputs.build_type }}
+  CC: ${{ inputs.compiler }}
+  CXX: ${{ inputs.compiler }}
```

Comment:
This isn't correct, but maybe it works because we are only using C files. But it should be `g++` or `clang++`

Reply:
You right, even though both compilers can automatically switch mode based on file extensions they still have differences against what stdlib program will be linked. However the only C++ code that we have is demos in libkmip and we don't compile it at all. So I will just drop CXX variable here.